clients/admin-ui/src/features/integrations/integration-type-info/rdsMySQLInfo.tsx

```diff
@@ -0,0 +1,85 @@
+import { ListItem } from "fidesui";
+
+import {
+  InfoHeading,
+  InfoText,
+  InfoUnorderedList,
+} from "~/features/common/copy/components";
+import ShowMoreContent from "~/features/common/copy/ShowMoreContent";
+import { ConnectionCategory } from "~/features/integrations/ConnectionCategory";
+import { AccessLevel, ConnectionType } from "~/types/api";
+
+export const RDS_MYSQL_PLACEHOLDER = {
+  name: "Amazon RDS MySQL",
+  key: "rds_mysql_placeholder",
+  connection_type: ConnectionType.RDS_MYSQL,
+  access: AccessLevel.READ,
+  created_at: "",
+};
+
+export const RDS_MYSQL_TAGS = [
+  "Database",
+  "Detection",
+  "Discovery",
+  "DSR automation",
+];
+
+export const RDSMySQLOverview = () => (
+  <>
+    <InfoHeading text="Overview" />
+    <InfoText>
+      Amazon RDS MySQL is a fully-managed relational database service that
+      simplifies the setup, maintenance, management, and administration of MySQL
+      databases. Connect Fides to your Amazon RDS MySQL to detect and track
+      changes in schemas and tables and automatically discover and label data
+      categories to proactively manage data governance risks.
+    </InfoText>
+    <ShowMoreContent>
+      <InfoHeading text="Categories" />
+      <InfoUnorderedList>
+        <ListItem>Database</ListItem>
+        <ListItem>NoSQL database</ListItem>
+        <ListItem>Storage system</ListItem>
+        <ListItem>Data detection</ListItem>
+        <ListItem>Data discovery</ListItem>
+        <ListItem>DSR automation</ListItem>
+      </InfoUnorderedList>
+      <InfoHeading text="Permissions" />
+      <InfoText>
+        For detecting database RDS instances and clusters, Fides requires an IAM
+        user with the following permissions/role:
+      </InfoText>
+      <InfoUnorderedList>
+        <ListItem>rds:DescribeDBClusters</ListItem>
+        <ListItem>rds:DescribeDBInstances</ListItem>
+        <ListItem>rds-db:connect</ListItem>
+      </InfoUnorderedList>
+      <InfoText>
+        And per database it requires the following permissions:
+      </InfoText>
+      <InfoUnorderedList>
+        <ListItem>
+          CREATE USER &apos;username_you_configured&apos; IDENTIFIED WITH
+          AWSAuthenticationPlugin AS &apos;RDS&apos;;
+        </ListItem>
+        <ListItem>
+          ALTER USER &apos;username_you_configured&apos;@&apos;%&apos; REQUIRE
+          SSL;
+        </ListItem>
+        <ListItem>
+          GRANT ALL PRIVILEGES ON database_you_configured.* TO
+          &apos;username_you_configured&apos;@&apos;%&apos;;
```

Comment:
Do we really need all privileges? Not doubting you here but I'd be better if we listed the individual privileges or added some justification why we need all privileges